app/routes/routes.py

```diff
@@ -0,0 +1,115 @@
+from flask import Blueprint, jsonify, request
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+from datetime import datetime 
```

Comment:
Optionally, you could name this entire file `task_routes.py` just to match the naming convention for the goal routes.